Cartfile

```diff
@@ -1 +1,2 @@
 github "PlayCover/PlayTools" "master"
+github "scinfu/SwiftSoup"
```

Comment:
This doesn't look like the right way to import a package? I mean there are other imported packages like `inject` and `Sparkle` but they're not written here. I think they're configured in the project configuration? I'm not sure what the difference is tho.

Reply:
Indeed, the dependency should be added through the Swift Package Manager. That way we keep Carthage exclusively for PlayTools. https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app

Reply:
Will try to do it, I already tried add it with package manager (same as Sparkle)  but build fail. Will try to fix it

Reply:
ok, I think I did it but my Carthage is bugged and I skipped the `Carthage Bootstrap` build phase but I did it with terminal. I think I have Cache somewhere but I removed all path I found

Reply:
I fixed this changing SwiftSoup version